Source/WebCore/platform/text/cocoa/LocaleCocoa.mm

```diff
@@ -90,11 +90,17 @@
     DateComponentsType type = dateComponents.type();
 
     ASSERT(type != DateComponentsType::Invalid);
-#if !ENABLE(INPUT_TYPE_WEEK_PICKER)
-    // "week" type not supported.
-    if (type == DateComponentsType::Week)
+
+    if (type == DateComponentsType::Week) {
+#if ENABLE(INPUT_TYPE_WEEK_PICKER)
+        // NSDateFormatter is not used here because it handles week numbering differently than ISO-8601.
+        return [NSString stringWithFormat:@"%@ %i, %i", (NSString *)inputWeekLabel(), dateComponents.week(), dateComponents.fullYear()];
```

Comment:
If we're going with this approach, it's probably better to make the localized string itself be a format string.

Reply:
Fixed in latest commit